tests/verifications/openai/fixtures/fixtures.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import os
+
+import pytest
+from openai import OpenAI
+
+
+@pytest.fixture
+def providers_model_mapping():
```

Comment:
later - will be good to have a separate yaml file per provider so that it's more easy to add new models etc. also yaml means comments can be added.